torch/package/package_exporter.py

```diff
@@ -112,6 +112,7 @@ def __init__(
         self.provided: Dict[str, bool] = {}
         self.verbose = verbose
         self.script_module_serializer = torch._C.ScriptModuleSerializer(self.zip_file)
+        self.gate_torchscript_serialization = True
```

Comment:
let's just make this a global with a leading underscore, and set it in the `setUp()` for the tests. It's fine to be a universal setting for now.